code/upaya/app/controllers/devise/account_type_controller.rb

```diff
@@ -43,7 +43,7 @@ def unconfirmed_representative?
     def success
       redirect_to dashboard_index_path,
                   notice: t('upaya.notices.account_created',
-                            date: (Time.now + 1.year).strftime('%B %d, %Y'))
+                            date: (Time.current + 1.year).strftime('%B %d, %Y'))
```

Comment:
I'm not too worried about timezones, but perhaps you can point out potential issues. The purpose of this particular notice is to let the user know that their password will expire a year from when they created their account (assuming we even want to keep this feature). 

We only show them the month, day, and year. Since `Time.current` is the same as `Time.zone.now`, and since we've set the app's timezone to Eastern, the only issue I see is that the date could be off by one day if someone on the west coast signs up at 9pm. 

My guess is that people will not be marking that date down on their calendar. The worst that can happen is that when they try to sign in (thinking their password hasn't expired yet), they'll be asked to update it.